platform/commonUI/edit/src/policies/EditableMovePolicy.js

```diff
@@ -39,8 +39,8 @@ define([], function () {
             selectedObject = context.selectedObject,
             key = action.getMetadata().key;
 
-        if (key === 'move' && domainObject.hasCapability('editor')) {
-            return !!selectedObject && selectedObject.hasCapability('editor');
+        if (key === 'move' && domainObject.hasCapability('editor') && domainObject.getCapability('editor').inEditContext()) {
+            return !!selectedObject && selectedObject.hasCapability('editor') && selectedObject.getCapability('editor').inEditContext();
```

Comment:
Line length of these conditions gets a little long, would like to break this up across multiple lines. Line breaks in an `if` condition are also a little funny-looking so maybe stick this in a function or assign it to a variable above?